syntax = "proto3";

package Protocol;

message BreakEquipItem{
	uint64 SlotIndex = 1;
}